Dockerfile

```diff
@@ -7,11 +7,12 @@ RUN \
   apk add --no-cache \
     dumb-init \
     git && \
-  go get github.com/gohugoio/hugo && \
+  go get github.com/kardianos/govendor && \
+  govendor get github.com/gohugoio/hugo && \
   cd $GOPATH/src/github.com/gohugoio/hugo && \
   go install && \
   cd $GOPATH && \
-  rm -rf pkg src && \
+  rm -rf pkg src bin/govendor && \
```

Comment:
hi @anthonyfok ,

thanks for the answers, 129MB would be a big optimization.
i had a look inside the container (https://hub.docker.com/r/ellerbrock/alpine-hugo/) but could not see any `.cache` folder.
Are there any new changes causing this or do i miss something?

`docker run -it --entrypoint ash ellerbrock/alpine-hugo`

```
/ $ cd /go
/go $ ls -alF
total 12
drwxrwxrwx    1 root     root          4096 Jul 21 16:39 ./
drwxr-xr-x    1 root     root          4096 Jul 23 06:20 ../
drwxrwxrwx    1 root     root          4096 Jul 21 16:39 bin/
/go $
```

Cheers Maik